### PR TITLE
Added misc stuff

### PR DESCRIPTION
**Pull request by RealKC**
- Adds @napstr's beloved lennies;
- Does more stuff about #126;
- made `;;s`(the shrug) be ;;`sh` because of the above.

### CODE REVIEW

**Comment by RealKC** [comment]
F, this is what I get for messing up SmartGit

File: FredBoat/src/main/java/fredboat/commandmeta/init/MusicCommandInitializer.java [hunk 1]
```diff
@@ -61,11 +61,11 @@ public static void initCommands() {
...
         
         /* Control */
-        CommandRegistry.registerCommand("play", new PlayCommand(SearchUtil.SearchProvider.YOUTUBE), "yt", "youtube");
+        CommandRegistry.registerCommand("play", new PlayCommand(SearchUtil.SearchProvider.YOUTUBE), "yt", "youtube", "p");
         CommandRegistry.registerCommand("sc", new PlayCommand(SearchUtil.SearchProvider.SOUNDCLOUD), "soundcloud");
-        CommandRegistry.registerCommand("skip", new SkipCommand(), "sk");
+        CommandRegistry.registerCommand("skip", new SkipCommand(), "sk", "s");
         CommandRegistry.registerCommand("join", new JoinCommand(), "summon", "jn");
-        CommandRegistry.registerCommand("leave", new LeaveCommand(), "lv");
+        CommandRegistry.registerCommand("leave", new LeaveCommand(), "lv", "s");
```
Comment (napstr) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MusicCommandInitializer.java]: This looks wrong.

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java [hunk 2]
```diff
@@ -102,21 +104,25 @@ public static void initCommands() {
         CommandRegistry.registerCommand("powerpoint", new RemoteFileCommand("http://i.imgur.com/i65ss6p.png"));
         
         /* Text Faces & Unicode 'Art' & ASCII 'Art' and Stuff */
-        CommandRegistry.registerCommand("s", new TextCommand("¯\\_(ツ)_/¯"), "shrug");
-        CommandRegistry.registerCommand("lenny", new TextCommand("( ͡° ͜ʖ ͡°)"));
+        CommandRegistry.registerCommand("sh", new TextCommand("¯\\_(ツ)_/¯"), "shrug");
```
Comment (napstr) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: I think sh was taken by the shuffle command.

**Comment by RealKC** [comment]
I think this should be fine now?

**Comment by Shredder121** [comment]
I've got one more idea, to clean up the constants.
If you put it in a `String.join("\n", ...)` call, you can't ever forget a newline.
```java
public static final String DOG = String.join("\n",
        "┈┈╱▏┈┈┈┈┈╱▔▔▔▔╲┈ ",
        "┈┈▏▏┈┈┈┈┈▏╲▕▋▕▋▏ ",
        "┈┈╲╲┈┈┈┈┈▏┈▏┈▔▔▔▆ ------- BORF!",
        "┈┈┈╲▔▔▔▔▔╲╱┈╰┳┳┳╯ ",
        "╱╲╱╲▏┈┈┈┈┈┈▕▔╰━╯ ",
        "▔╲╲╱╱▔╱▔▔╲╲╲╲┈┈┈ ",
        "┈┈╲╱╲╱┈┈┈┈╲╲▂╲▂┈ ",
        "┈┈┈┈┈┈┈┈┈┈┈╲╱╲╱┈",
        "Spread him to take over Discord!(Don't spam him though!)"
);
//dog created by @Ace#8925
```
(then remove all the explicit `\n`s)

File: FredBoat/src/main/java/fredboat/util/AsciiArtConstant.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ */
+public final class AsciiArtConstant {
+
+    public static final String MAGICAL_LENNY = String.join("\n",
+            "（ ͡° ͜ʖ ͡°)つ━☆・。\n",
+            "⊂　　 ノ 　　　・゜+.\n",
+            "　しーＪ　　　°。+ ´¨)\n",
+            "　　　　　　　　　.· ´¸.·´¨) ¸.·*¨)\n",
+            "　　　　　　　　　　(¸.·´ (¸.·' ☆ ABRA KADABRA..."
+    );
```
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/util/AsciiArtConstant.java]: Still `\n`s...

File: FredBoat/src/main/java/fredboat/util/AsciiArtConstant.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    public static final String EAGLE_OF_LENNY = String.join("\n",
+            ":＜￣｀ヽ、　　　　　　　／￣＞\n",
+            "　ゝ、　　＼　／⌒ヽ,ノ 　/´\n",
+            "　　　ゝ、　`（ ( ͡° ͜ʖ ͡°) ／\n",
+            "　　 　　>　 　 　,ノ\n",
+            "　　　　　∠_,,,/´”"
+    );
```
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/util/AsciiArtConstant.java]: Still `\n`s...

File: FredBoat/src/main/java/fredboat/util/AsciiArtConstant.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            "┈┈╱▏┈┈┈┈┈╱▔▔▔▔╲┈ \n",
+            "┈┈▏▏┈┈┈┈┈▏╲▕▋▕▋▏ \n",
+            "┈┈╲╲┈┈┈┈┈▏┈▏┈▔▔▔▆ ------- BORF!\n",
+            "┈┈┈╲▔▔▔▔▔╲╱┈╰┳┳┳╯ \n",
+            "╱╲╱╲▏┈┈┈┈┈┈▕▔╰━╯ \n",
+            "▔╲╲╱╱▔╱▔▔╲╲╲╲┈┈┈ \n",
+            "┈┈╲╱╲╱┈┈┈┈╲╲▂╲▂┈ \n",
+            "┈┈┈┈┈┈┈┈┈┈┈╲╱╲╱┈\n",
+            "Spread him to take over Discord!(Don't spam him though!)"
+   );
```
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/util/AsciiArtConstant.java]: Still `\n`s...

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java [hunk 6]
```diff
@@ -102,21 +104,25 @@ public static void initCommands() {
...
         CommandRegistry.registerCommand("sendenergy", new TextCommand("༼ つ ◕_◕ ༽つ"));
-        CommandRegistry.registerCommand("peeking", new TextCommand("┬┴┬┴┤ ͜ʖ ͡°) ├┬┴┬┴"), "peekinglenny", "peek");
         CommandRegistry.registerCommand("dealwithit", new TextCommand("(•\\_•) ( •\\_•)>⌐■-■ (⌐■_■)"), "dwi");
         CommandRegistry.registerCommand("channelingenergy", new TextCommand("(ﾉ◕ヮ◕)ﾉ*:･ﾟ✧ ✧ﾟ･: *ヽ(◕ヮ◕ヽ)"));
         CommandRegistry.registerCommand("butterfly", new TextCommand("Ƹ̵̡Ӝ̵̨̄Ʒ"));
         CommandRegistry.registerCommand("angrytableflip", new TextCommand("(ノಠ益ಠ)ノ彡┻━┻"), "tableflipbutangry", "atp");
-        CommandRegistry.registerCommand("cooldog", new DogCommand(), "dog", "dogmeme");
-        CommandRegistry.registerCommand("lood", new TextCommand("T-that's l-lewd, baka!!!"), "lewd", "l00d");
+        CommandRegistry.registerCommand("dog", new TextCommand(DOG), "cooldog", "dogmeme");
+        CommandRegistry.registerCommand("loqod", new TextCommand("T-that's l-lewd, baka!!!"), "lewd", "l00d");
```
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: What's `loqod`? 🤔

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java [hunk 7]
```diff
@@ -34,6 +34,8 @@
 import fredboat.command.util.*;
 import fredboat.commandmeta.CommandRegistry;
 
+import static fredboat.util.AsciiArtConstant.*;
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Please no `import static` aboose 👀

Comment (RealKC) [reply FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Well, without that I would've had to write AsciiArtConstant.<whatever string I needed> every time I needed a string from within there, which is kind of annoying.(Unless I'm a total derp.)

Comment (Frederikam) [reply FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Yes, and that makes a lot more sense than, say, just typing `DOG`

Comment (RealKC) [reply FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: I guess.

**Comment by Shredder121** [comment]
Addressed.

**Comment by Shredder121** [comment]
Shall I squash the commits that KC did as a user without email?
Or will we squash the commits anyway?

**Comment by Frederikam** [comment]
We will squash after @napstr has approved the changes

**Pull request merged**